src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.tsx

```diff
@@ -83,10 +82,9 @@ const EditableSectionBlocks = (props: EditableSectionBlockProps) => {
           <div
             key={block.id}
             id={`editableSectionBlocks-${block.id}`}
-            className={classNames(
-              'govuk-!-margin-bottom-9',
-              openedCommentId.includes(block.id) && styles.openSectionBlock,
-            )}
+            className={classNames('govuk-!-margin-bottom-9', {
+              openSectionBlocks: openedCommentIds.includes(block.id),
```

Comment:
Hey @HiveIT-Lewis , should this be `openSectionBlock` rather than `openSectionBlocks`?